ci.fmf

```diff
@@ -0,0 +1,27 @@
+/test/build:
+    /smoke:
+      execute:
+        how: shell
+        script:
+        - packit --version
+        - packit --help
+        - packit status https://github.com/packit-service/packit
+        - packit status https://src.fedoraproject.org/rpms/packit
```

Comment:
I thought we need an API token for this

Reply:
I was suprised as well. We are calling only publicly-accessible urls.